rskj-core/src/main/java/org/ethereum/net/rlpx/MessageCodec.java

```diff
@@ -141,9 +141,7 @@ private Message decodeMessage(ChannelHandlerContext ctx, List<Frame> frames) thr
 
         Message msg = createMessage((byte) frameType, payload);
 
-        if (loggerNet.isInfoEnabled()) {
-            loggerNet.info("From: \t{} \tRecv: \t{}", channel, msg.toString());
-        }
+        loggerNet.trace("From: \t{} \tRecv: \t{}", channel, msg.toString());
```

Comment:
I would remove this `.toString()` b/c this call will be made even if the TRACE level is not enabled

Reply:
:+1: